audio_service/android/src/main/java/com/ryanheise/audioservice/AudioServicePlugin.java

```diff
@@ -401,15 +484,6 @@ public void onMethodCall(MethodCall call, final Result result) {
                     if (AudioService.instance != null) {
                         AudioService.instance.configure(config);
                     }
-                    mainClientInterface = ClientInterface.this;
-                    if (audioHandlerInterface == null) {
-                        audioHandlerInterface = new AudioHandlerInterface(messenger, true /*androidEnableQueue*/);
-                        AudioService.init(audioHandlerInterface);
-                    } else if (audioHandlerInterface.messenger != messenger) {
-                        // We've detected this is the real engine hosting the AudioHandler,
-                        // so update AudioHandlerInterface to connect to it.
-                        audioHandlerInterface.switchToMessenger(messenger);
-                    }
```

Comment:
I removed this, because it doesn't seem to be working for me.

Image we have to clients A and B:
1. A calls configure and creates an audio handler
2. B then also calls configure and then the handler is switched to him
3. A stops receiving any events until he calls the configure once more

If we wanna keep this functionally, we really need to unify the channels and just have a list them

By the way `mainClientInterface` also seems to be off, why when activity attaches, it becomes main? Why can't we have multiple of them?

Reply:
Actually I was wrong, I now understand how this works, gonna bring this back